apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -180,128 +179,135 @@ const ProfileView = () => {
     [ErrorCode.ThirdPartyIdentityProviderEnabled]: t("account_created_with_identity_provider"),
   };
 
-  if (isLoading || !user || isLoadingAvatar || !avatar)
-    return <SkeletonLoader title={t("profile")} description={t("profile_description")} />;
+  const isPageLoading = isLoading || !user || isLoadingAvatar || !avatar;
 
   const defaultValues = {
-    username: user.username || "",
-    avatar: avatar.avatar || "",
-    name: user.name || "",
-    email: user.email || "",
-    bio: user.bio || "",
+    username: user?.username || "",
```

Comment:
By this point user shouldn't be undefined right?

```suggestion
    username: user.username || "",
```

Reply:
Yeah you are right but typescript is complaining that the user is `undefined`. Previously the skeleton was rendered before the `Defaultvalue` so it didn't complain. now the loading check is inside the return statement  so it's complaining.

Reply:
has this been resolved? @G3root

Reply:
@PeerRich i think this is resolved. by the above point the user should be defined but typescript is throwing an error stating that the user is undefined, so i added a `optional chaining` operator additionally.